bpf/bpf_host.c

```diff
@@ -378,7 +379,14 @@ handle_ipv6_cont(struct __ctx_buff *ctx, __u32 secctx, const bool from_host,
 	info = lookup_ip6_remote_endpoint(dst, 0);
 
 #ifdef TUNNEL_MODE
-	if (info && info->flag_skip_tunnel)
+	__u32 src_subnet_id = lookup_ip6_subnet((union v6addr *)&ip6->saddr);
+	__u32 dst_subnet_id = lookup_ip6_subnet((union v6addr *)&ip6->daddr);
```

Comment:
This two extra lookup is only required for hybrid mode. Then, I think we should guard this with `if(CONFIG(<config name for hybrid mode>))`. Otherwise, we always need to pay lookup cost in the non-hybrid tunnel mode.

Reply:
Good idea. Made the change.